contrib/vexecutor/vcheck.h

```diff
@@ -39,6 +39,7 @@ typedef struct VectorizedState
 {
 	bool vectorized;
 	PlanState *parent;
+    bool* proj;
```

Comment:
Because this variable is only related to ao table, or has different content based on different table format. It's better to rename it to a more readable name, aoprojs or just use (void * opaque).